.gitignore

```diff
@@ -0,0 +1,5 @@
+.pyc
```

Comment:
Suggest squashing these last two commits together. I _think_ .pyc doesn't work -- or at least it didn't for me. But **/__pycache__/ should anyway result in all .pyc files being ignored.

For the rest, sorry, I wasn't clear about my example. /static/ and /log/ I would drop, those are a bit specific to my own Wagtail setup. The .sqlite3 file you could choose to ignore, but I think for the comparison tool you're probably better off keeping it in source control. I removed it from git in my own Wagtail because it updates every time I log in to the admin interface, so it collects a lot of unimportant changes. But for you I think it is just a wrapper, so perhaps better to leave in place?